spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -128,17 +107,16 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private RabbitExtendedBindingProperties extendedBindingProperties = new RabbitExtendedBindingProperties();
 
-	public RabbitMessageChannelBinder(ConnectionFactory connectionFactory, RabbitProperties rabbitProperties) {
-		super(true, new String[0]);
+	RabbitExchangeQueueProvisioner provisioningProvider;
```

Comment:
`private final`
Although I still wonder if we can make a super one as a `protected` and generic do not have this local copy as well...